diff --git a/devices/devices.go b/devices/devices.go
index cf97633..f916bff 100644
--- a/devices/devices.go
+++ b/devices/devices.go
@@ -21,7 +21,7 @@ var (
 func LoadSSDPservices(delay int) (map[string]string, error) {
 	// Reset device list every time we call this.
 	urlList := make(map[string]string)
-	list, err := ssdp.Search(ssdp.All, delay, "")
+	list, err := ssdp.Search(ssdp.All, delay, "0.0.0.0:1901")
 	if err != nil {
 		return nil, fmt.Errorf("LoadSSDPservices search error: %w", err)
 	}
